service/src/main/java/bio/terra/profile/db/ProfileDao.java

```diff
@@ -109,6 +121,18 @@ public List<BillingProfile> listBillingProfiles(int offset, int limit, Collectio
     return jdbcTemplate.query(SQL_LIST, params, new BillingProfileMapper());
   }
 
+  public List<String> listManagedResourceGroupsInSubscription(UUID tenantId, UUID subscriptionId) {
```

Comment:
Is it necessary to limit to a specific tenant?

Reply:
It is not, I thought it was possible for subscriptions in different tenants to share the same ID, but subscription IDs are globally unique, so I'll remove the tenant ID here and there's a few test checks that can be modified/removed as well